backend/tests/v5/unit/models/tickets.test.js

```diff
@@ -176,20 +203,21 @@ const testUpdateTicket = () => {
 		const author = generateRandomString();
 		const propToUpdate = generateRandomString();
 
-		test('should update the ticket to set properties', async () => {
+		test('should update the ticket to set properties (Container)', async () => {
```

Comment:
Not sure if i missed it but I don't see a Federation version?

This is probably  N/A anyway with the suggested change in the source code